build/teamcity-jepsen.sh

```diff
@@ -72,8 +72,10 @@ for test in "${tests[@]}"; do
         ssh -o "ServerAliveInterval=60" -o "StrictHostKeyChecking no" -i "$HOME/.ssh/${KEY_NAME}" "ubuntu@${controller}" "${testcmd}"
         if [ $? -ne 0 ]; then
             # Test failed: grab everything.
-            echo "Test failed. Grabbing logs..."
-            scp -o "StrictHostKeyChecking no" -ri "$HOME/.ssh/${KEY_NAME}" "ubuntu@${controller}:jepsen/cockroachdb/store/latest" "${artifacts_dir}"
+            echo "Test failed. Grabbing all logs..."
+            archive_path="jepsen/cockroachdb/store/failure-logs.tgz"
+            ssh -o "StrictHostKeyChecking no" -ri "$HOME/.ssh/${KEY_NAME}" "ubuntu@${controller}" "tar -czf ${archive_path} jepsen/cockroachdb/store/latest"
```

Comment:
On a sufficiently recent version of `tar` (perhaps with a separate tool installed), capital `J` (the `xz` format) generally has the best compression ratio/speed combination. But I don't know how widely supported it is so I'm fine sticking with `gzip`.